composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "thenetworg/oauth2-azure",
+    "name": "qbnk/oauth2-azure",
```

Comment:
I don't think that this change is intended. Please remove it. Thanks!

Reply:
Thank you, I've updated the PR! @hajekj